IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,212 @@
+@inherits CaptainCoder.Page
+@page "/activities/setting-builder"
+
+<Title>Setting Builder</Title>
+
+<Section Title="Introduction">
+
+     <Adventure Portrait="@Portraits.Ryan_1">
+        <Item>
+            <AdventureDialog>
+                <Text>
+                    Hello! I am Ryan, one of Captian Coder's students! I see that you are
+                    on your way to becoming a fully fledged program just like me! Maybe
+                    you can help me with this code I've been struggling with.
+                </Text>
+            </AdventureDialog>
+        </Item>
+        <Item>
+            <AdventureDialog>
+                <Text>
+                    Captian Coder tasked me with helping him plan out building a city for
+                    all his students to be able to practice coding. A place where we could
+                    share information and teach each other how to be better coders.
+                </Text>
+            </AdventureDialog>
+        </Item>
+        <Item>
+            <AdventureDialog>
+                <Text>
+                    The only problem is that building a city is hard! I can never seem
+                    to remember all the steps involved or keep track of all the details!
+                    What would be really helpful is if we could make a program that could
+                    keep track of all these details!
+                </Text>
+            </AdventureDialog>
+        </Item>
+        <Item>
+            <AdventureDialog>
+                <Text>
+                    In the following activity you are going to create a program that will help me plan
+                    out a city or town. You are going to need to prompt the user for information
+                    and be able to save that information in a variable using <Code>Console.WriteLine</Code>,
+                    <Code>Console.ReadLine</Code>, and the basics of <Code>variables</Code>.
+                </Text>
+            </AdventureDialog>
+        </Item>
+        <Item>
+            <AdventureDialog>
```

Comment:
Use the `Page` field in `AdventureDialog` to add in a "smart" button which will allow the user to add the pages if they are missing. You will have to break this into two Dialogs, one for each.